react/FileInput/styles.styl

```diff
@@ -0,0 +1,2 @@
+.parent
```

Comment:
I think it would be best with a more explicit classname such as `c-input-file`, for consistency.